scaleway/docs-content
# feat(bash): update frontmatter WDOC-205

## Udpate your docs project :
1. `cd ..` ( go on docs folder, not docs/content) 
2. `git pull && git status` --> check everything clean

## Test develop
Each time check title (h1) and description (only home/faq/faqs)  in the page

0. `yarn clean && yarn develop:algolia`
1. Go to [http://localhost:8003/](http://localhost:8003/)
2. Go to [http://localhost:8003/components](http://localhost:8003/components)
3. Go to [http://localhost:8003/compute/instances/api-cli/creating-managing-instances-with-cliv2/](http://localhost:8003/compute/instances/api-cli/creating-managing-instances-with-cliv2/)
4. Go to [http://localhost:8003/compute/instances/api-cli/](http://localhost:8003/compute/instances/api-cli/)
5. Go to [http://localhost:8003/tutorials/](http://localhost:8003/tutorials/)
6. Go to [http://localhost:8003/tutorials/create-valheim-server/](http://localhost:8003/tutorials/create-valheim-server/)
7. Go to [http://localhost:8003/faq/](http://localhost:8003/faq/)
8. Go to [http://localhost:8003/faq/bmaas](http://localhost:8003/faq/bmaas)
